vpr/src/place/place.cpp

```diff
@@ -3004,3 +3072,10 @@ static void init_annealing_state(t_annealing_state* state,
 bool placer_needs_lookahead(const t_vpr_setup& vpr_setup) {
     return (vpr_setup.PlacerOpts.place_algorithm == PATH_TIMING_DRIVEN_PLACE);
 }
+
+//transforms the vector moved_blocks to a vector of ints and adds it in current_info_p
+void transform_blocks_affected(t_pl_blocks_to_be_moved blocksAffected) {
+    current_info_p.blocks_affected_vector.clear();
+    for (size_t i = 0; i < blocksAffected.moved_blocks.size(); i++)
+        current_info_p.blocks_affected_vector.push_back(size_t(blocksAffected.moved_blocks[i].block_num));
```

Comment:
Why do you need the size_t conversion?  Whatever type block_num is should be OK (I think); if not comment why you need to convert.

Reply:
resolved

Reply:
the conversion is needed because block_num is of type ClusterBlockId and that type isn't recognized in expr_eval so we need a vector of ints. Added this explanation in the comments. So resolved.